sysutils/synkron/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           qmake5 1.0
+
+name                synkron
+version             1.6.2+r200
+revision            0
+
+categories          sysutils archivers
+platforms           darwin
+license             GPL-2+
+maintainers         nomaintainer
+
+description         Synkron is an application for synchronising folders
+long_description    Synkron is an application that helps you keep your files and folders always \
+                    updated. You can easily sync your documents, music or pictures to have their \
+                    latest versions everywhere.
+
+homepage            http://synkron.sourceforge.net/
+master_sites        sourceforge
+
+fetch.type          svn
+svn.url             https://svn.code.sf.net/p/synkron/code/trunk
+svn.revision        200
+
+qt5.min_version     5.0
+qt5.depends_build_component \
+                    qttools
+
+post-extract {
+    delete {*}[glob ${worksrcpath}/i18n/Synkron-*.qm]
+}
+
+patch.pre_args      -p1
+patchfiles          patch-add_missing_translations.diff
+
+post-patch {
+    exec sed -i -e "/QMAKE_MAC_SDK/d" ${worksrcpath}/Synkron.pro
+    exec sed -i -e "/QMAKE_MACOSX_DEPLOYMENT_TARGET/d" ${worksrcpath}/Synkron.pro
+}
```

Comment:
this should be done via a `reinplace` instead of doing `exec`

Reply:
@reneeotten, done